.github/ISSUE_TEMPLATE/release.md

```diff
@@ -60,7 +71,7 @@ Note: Do not publish draft releases from PR branches, because those branches wil
 
 ### Burn In
 
-Ensure that Manta DevOps has run the new release on Como and Baikal nodes
+Ensure that Manta DevOps has run the new release on Baikal nodes
```

Comment:
12 hours won't be enough. we really shouldn't clear a release that hasn't been running without anomalies for at least 2 days, this RAM leak only became apparent on baikal after 2 days

Reply:
also this is duplicated with line 38 https://github.com/Manta-Network/Manta/pull/715/files#diff-4482ca0941ca8f0e3db46d20770ecf9158cc00322eee6e760460c47a4a5932c2R38

so maybe remove this

Reply:
Yeah let's make it 3 days, not sure where this 12 hours was coming from.